modules/openapi-generator/src/main/resources/swift5/modelObject.mustache

```diff
@@ -76,16 +76,17 @@
     // Decodable protocol methods
 
     {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}}{{#objcCompatible}} required{{/objcCompatible}} init(from decoder: Decoder) throws {
-        let container = try decoder.container(keyedBy: String.self)
+        let container = try decoder.container(keyedBy: CodingKeys.self)
```

Comment:
@aymanbagabas could you please explain what the changes to the function `init(from decoder: Decoder) throws` do, or what do they fix?